rxbinding-support-recyclerview/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<manifest package="com.jakewharton.rxbinding.support.v7"
```

Comment:
I was just following the convention I saw in other tests, so I'll check if it's necessary